app/models/event_stream.rb

```diff
@@ -64,29 +72,29 @@ def self.event_groups
 
   # TODO: Consider moving since this is EmsEvent specific. group, group_level and group_name exposed as a virtual columns for reports/api.
   def self.group_and_level(event_type)
-    level = :detail # the level is detail as default
+    level = DEFAULT_GROUP_LEVEL # the level is detail as default
     egroups = event_groups
 
     group = egroups.detect do |_, value|
-      GROUP_LEVELS
+      group_levels
         .detect { |lvl| value[lvl]&.any? { |typ| !typ.starts_with?("/") && typ == event_type } }
         .tap { |level_found| level = level_found || level }
     end&.first
 
     group ||= egroups.detect do |_, value|
-      GROUP_LEVELS
+      group_levels
         .detect { |lvl| value[lvl]&.any? { |typ| typ.starts_with?("/") && Regexp.new(typ[1..-2]).match?(event_type) } }
         .tap { |level_found| level = level_found || level }
     end&.first
 
-    group ||= :other
+    group ||= DEFAULT_GROUP_NAME
     return group, level
   end
 
   def self.group_name(group)
     return if group.nil?
     group = event_groups[group.to_sym]
-    group.nil? ? 'Other' : group[:name]
+    group.nil? ? DEFAULT_GROUP_NAME.to_s.capitalize : group[:name]
```

Comment:
Not sure if this will break I18N - we may have to figure that out in a follow up.

Reply:
Yeah, I don't know how to test it.  I can ask around and do it in a followup.